dist/index.html

```diff
@@ -10,26 +10,15 @@
 <script defer src="/runtime.bundle.js"></script><script defer src="/index.bundle.js"></script></head>
 <body>
     <div class="todo-list">
-    <form action="">
-        <div class="input-area"><input class="input" id="input-todo" type="text" placeholder="Add to your list..."><i class="fa fa-arrow-right" id="enter"></i></div>
-        <ul class="todolists">
-            <!--<li class="chk01">
-                <div class="chk02">
-                    <input type="checkbox">
-                    <label>Eat</label>
-                </div>
-                <i class="fas fa-trash"></i>
-            </li>
-            <li class="chk01">
-                <div class="chk02">
-                    <input type="checkbox">
-                    <label>Eat</label>
-                </div>
-                <i class="fa fa-ellipsis-v"></i>
-            </li>-->
-        </ul>
-        <button type="submit" class="clear-btn" id="btnadd">Clear all completed</button>
-    </form>
+        <div class="head">
+            <h1>Today's To Do</h1>
+            <i class="fa fa-sync-alt"></i>
+        </div>
+        <form action="">
+            <div class="input-area"><input class="input" id="input-todo" type="text" placeholder="Add to your list..."><i class="fa fa-arrow-right" id="enter"></i></div>
+            <ul class="todolists"></ul>
+            <button type="submit" class="clear-btn" id="btnadd">Clear all completed</button>
+        </form>
```

Comment:
- [ ] I can see that you made a nice design on this project but we are missing something for it to look more attractive to recruiters and users, kindly fix this for your project to match the design given to this project :wink: 
Your Desing 
![Screenshot from 2023-02-23 18-54-18](https://user-images.githubusercontent.com/88894525/220976667-5111b001-028a-4c5d-8c0d-99c4acabbfb6.png)

This is how it should display :point_down:  try this please 
![Screenshot from 2023-02-23 18-53-47](https://user-images.githubusercontent.com/88894525/220977022-d7f17ef2-f522-4268-b915-be8f1634b7b5.png)

Reply:
@Alfredbis29 Kindly check again how you have set up the application, i am able to add the tasks by clicking the arrow icon.